pytorch/training/docker/1.13/py3/Dockerfile.cpu

```diff
@@ -196,17 +215,6 @@ ARG PT_TORCHDATA_URL
 RUN pip uninstall -y torch torchvision torchaudio torchdata \
  && pip install --no-cache-dir -U ${PT_EC2_TRAINING_URL} ${PT_TORCHVISION_URL} ${PT_TORCHAUDIO_URL} ${PT_TORCHDATA_URL}
 
-RUN mkdir -p /etc/pki/tls/certs && cp /etc/ssl/certs/ca-certificates.crt /etc/pki/tls/certs/ca-bundle.crt
-
-RUN HOME_DIR=/root \
```

Comment:
lets keep this as is, we need to run compliance at the end for each image (ec2/sm) so that it generates file licenses for all the packages in that dockerfile section